docs/docs/porting-create-react-app-to-gatsby.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Porting Create React App to Gatsby
+---
+
+## What is Create React App?
+
+Create React App is an officially supported tool from React to setup React apps, without having to deal with complicated configurations. It provides a default setup for tools like webpack and Babel that are useful in a modern development pipeline.
+
+Gatsby is similar in that it can also help you setup an app and removes much of the configuration headache, but offers some additional advantages like the ability to leverage Gatsby's ecosystem and performance optimizations. React's own [docs](https://reactjs.org/) run on Gatsby, and React even [recommends Gatsby](https://github.com/facebook/create-react-app#popular-alternatives) to users of Create React App!
```

Comment:
```suggestion
Gatsby is similar in that it can also help you set up an app and removes much of the configuration headache, but offers some additional advantages like the ability to leverage Gatsby's ecosystem and performance optimizations. React's own [docs](https://reactjs.org/) run on Gatsby, and React even [recommends Gatsby](https://github.com/facebook/create-react-app#popular-alternatives) to users of Create React App!
```